example/src/Main.purs

```diff
@@ -72,7 +72,7 @@ ui = parentComponent render eval
                         )
                         Nothing
                 , initialState :
-                    { key : Nothing, editor : Nothing }
+                    initialAceState
```

Comment:
Ah right, you just wrap a `SlotConstructor` in a `Slot`, which is one of the `HTML` data constructors. The `slot` version is a smart constructor that makes the `SlotConstructor`-in-`Slot` for you.